mne/tests/test_source_estimate.py

```diff
@@ -550,63 +549,64 @@ def test_morph_data():
     stc_to.crop(0.09, 0.1)  # for faster computation
     assert_array_equal(stc_to.time_as_index([0.09, 0.1], use_rounding=True),
                        [0, len(stc_to.times) - 1])
-    pytest.raises(ValueError, stc_from.morph, subject_to, grade=3, smooth=-1,
-                  subjects_dir=subjects_dir)
-    stc_to1 = stc_from.morph(subject_to, grade=3, smooth=12, buffer_size=1000,
-                             subjects_dir=subjects_dir)
+
+    morph = SourceMorph(subject_from=subject_from,
+                        subject_to=subject_to,
+                        spacing=5,
+                        smooth=-1,
+                        subjects_dir=subjects_dir)
+    # negative smooth
+    pytest.raises(ValueError, morph, stc_from)
+    stc_to1 = SourceMorph(subject_to=subject_to,
+                          spacing=3,
+                          smooth=12,
+                          subjects_dir=subjects_dir)(stc_from)
+
     stc_to1.save(op.join(tempdir, '%s_audvis-meg' % subject_to))
     # Morphing to a density that is too high should raise an informative error
     # (here we need to push to grade=6, but for some subjects even grade=5
     # will break)
-    pytest.raises(ValueError, stc_to1.morph, subject_from, grade=6,
-                  subjects_dir=subjects_dir)
+    morph = SourceMorph(subject_from=subject_to,
+                        subject_to=subject_from,
+                        spacing=6,
+                        subjects_dir=subjects_dir)
+    pytest.raises(ValueError, morph, stc_to1)
     # make sure we can specify vertices
     vertices_to = grade_to_vertices(subject_to, grade=3,
                                     subjects_dir=subjects_dir)
-    stc_to2 = morph_data(subject_from, subject_to, stc_from,
-                         grade=vertices_to, smooth=12, buffer_size=1000,
-                         subjects_dir=subjects_dir)
-    # make sure we can use different buffer_size
```

Comment:
@TommyClausner you removed this test as you got rid of the buffer_size parameter that was used in combination with n_jobs?

@larsoner what's your take on this? You had done this in the first place.

Reply:
We shouldn't need `buffer_size` because we (should) always precompute now

Reply:
:D Yeah I did at some point - no clue how it sneaked in again - can indeed be deleted